src/github.com/getlantern/flashlight/client/balancer.go

```diff
@@ -37,7 +38,10 @@ func (client *Client) initBalancer(cfg *ClientConfig) (*balancer.Balancer, error
 			log.Debugf("Adding chained server: %v", s.Addr)
 			dialers = append(dialers, dialer)
 		} else {
-			log.Errorf("Unable to configure chained server. Received error: %v", err)
+			elog.Log(err, errlog.WithOp("configure"), errlog.WithProxy(&errlog.ProxyingInfo{
```

Comment:
You could improve the syntax by either providing a convenience method on ChainedServerInfo to create the ProxyingInfo, or you could actually make ProxyingInfo an interface and make sure that ChainedServerInfo implements that interface.  Both options make life for the user of this API easier.